### PR TITLE
DOCS-319 - Alerts/Monitors tag update

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Alert/Monitor tag update. As of 9/5, still on hold per @jinkane.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/DOCS-319

### CODE REVIEW

File: docs/alerts/monitors/alert-response.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ The following is an example Slack payload with the variable:
...
 
-<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Alerts list, click the bell icon in the top menu. <br/> <img src={useBaseUrl('img/alerts/alert-list-page-bell-border.png')} alt="alert-list-page-bell-border" width="300"/> 
-<!--Kanso 
-[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Alerts list, in the main Sumo Logic menu select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**. 
+<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Alerts list, click the bell icon in the top menu. <br/> <img src={useBaseUrl('img/alerts/alert-list-page-bell-border.png')} alt="alert-list-page-bell-border" width="300"/>
+<!--Kanso
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Alerts list, in the main Sumo Logic menu select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**.
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-response.md:70]:
```suggestion
<!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Alerts list, click the bell icon in the top menu. <br/> <img src={useBaseUrl('img/alerts/alert-list-page-bell-border.png')} alt="alert-list-page-bell-border" width="300"/> 
<!--Kanso 
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Alerts list, in the main Sumo Logic menu select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**.
```

File: docs/alerts/monitors/alert-response.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ The Alerts list shows all of your Alerts from Monitors triggered within the pas
 
 <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Alerts list, click the bell icon in the top menu. <br/> <img src={useBaseUrl('img/alerts/alert-list-page-bell-border.png')} alt="alert-list-page-bell-border" width="300"/> 
 <!--Kanso 
-[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Alerts list, in the main Sumo Logic menu select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**. 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Alerts list, in the main Sumo Logic menu select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**.
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-response.md:70]:
```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Alerts list, in the main Sumo Logic menu select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**. 
```

File: docs/alerts/monitors/settings.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ description: Monitors continuously query your logs or metrics and send notificat
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The Monitors page allows you to view, create, manage, and organize your Monitors. 
+The Monitors page allows you to view, create, manage, and organize your Monitors.
 
 <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Monitors page, in the main Sumo Logic menu select **Manage Data > Monitoring > Monitors**.  
-<!--Kanso 
-[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. 
+<!--Kanso
```
Comment (kimsauce) [inline docs/alerts/monitors/settings.md:13]:
```suggestion
<!--Kanso 
```

File: docs/alerts/monitors/settings.md [hunk 4]
```diff
@@ -7,11 +7,11 @@ description: Monitors continuously query your logs or metrics and send notificat
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The Monitors page allows you to view, create, manage, and organize your Monitors. 
+The Monitors page allows you to view, create, manage, and organize your Monitors.
```
Comment (kimsauce) [inline docs/alerts/monitors/settings.md:10]:
```suggestion
The Monitors page allows you to view, create, manage, and organize your Monitors. 
```

File: docs/alerts/monitors/settings.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ The Monitors page allows you to view, create, manage, and organize your Monitors
 
 <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> To access the Monitors page, in the main Sumo Logic menu select **Manage Data > Monitoring > Monitors**.  
 <!--Kanso 
-[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. 
+[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**.
```
Comment (kimsauce) [inline docs/alerts/monitors/settings.md:14]:
```suggestion
[**New UI**](/docs/get-started/sumo-logic-ui-new/). To access the Monitors page, in the main Sumo Logic menu select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. 
```

File: docs/alerts/monitors/settings.md [hunk 6]
```diff
@@ -115,6 +117,19 @@ To add a tag while creating a new monitor:
    * Apply an existing tag by selecting the key and its possible values from the dropdown.
 1. Click **Save**.
 
+
+### What happens next?
+
+Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.
+
+To view and edit tags, go to your alerts list page at **Manage Data > Monitoring > Monitors**, where you'll see a tags column. This page allows filtering by tags.
```
Comment (jpipkin1) [inline docs/alerts/monitors/settings.md:125]:
```suggestion
To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->

Here you'll see a tags column. This page allows filtering by tags.
```

File: docs/alerts/monitors/settings.md [hunk 7]
```diff
@@ -115,6 +117,19 @@ To add a tag while creating a new monitor:
...
+
+<img src={useBaseUrl('img/alerts/monitors-alerts-list-tags.png')} alt="Alerts list page showing a column labeled 'Tags' next to alert details" />
+
+You will also find your tags on the Alert Response page. Here's an example:
+
+<img src={useBaseUrl('img/alerts/alert-response-tags.png')} alt="Alert Response page displaying graphs with tags shown in the tags section" />
```
Comment (jpipkin1) [inline docs/alerts/monitors/settings.md:131]:
```suggestion
<img src={useBaseUrl('img/alerts/monitors-alerts-list-tags.png')} alt="Alerts list page showing a column labeled 'Tags' next to alert details" style={{border: '1px solid gray'}} width="800"  />

You will also find your tags on the Alert Response page. Here's an example:

<img src={useBaseUrl('img/alerts/alert-response-tags.png')} alt="Alert Response page displaying graphs with tags shown in the tags section" style={{border: '1px solid gray'}} width="800" />
```

**Review by jpipkin1 — APPROVED** [review]
@kimsauce, I have only a couple of minor suggestions.

**Comment by kimsauce** [comment]
> @kimsauce, I have only a couple of minor suggestions.

thank you, @jpipkin1 !

File: docs/alerts/monitors/settings.md [hunk 8]
```diff
@@ -115,6 +117,21 @@ To add a tag while creating a new monitor:
    * Apply an existing tag by selecting the key and its possible values from the dropdown.
 1. Click **Save**.
 
+
+### What happens next?
+
+Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.
+
+To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->
```
Comment (jinkane) [inline docs/alerts/monitors/settings.md:125]: navigating via `Manage Data > Monitoring > Monitors` will take the user to the Monitors Page and not the Alert List page as we have mentioned in the previous line.

Comment (jinkane) [inline docs/alerts/monitors/settings.md:125]: Also users cannot edit the tags on the Alerts. They can only see it and use them for filtering.

Although users can edit tags on the Monitor, the edited tags will not be updated on the alerts that were previously triggered. We should mention this too.

**Review by jinkane — COMMENTED** [review]
Please address the comments.

File: docs/alerts/monitors/settings.md [hunk 9]
```diff
@@ -115,6 +117,21 @@ To add a tag while creating a new monitor:
    * Apply an existing tag by selecting the key and its possible values from the dropdown.
 1. Click **Save**.
 
+
+### What happens next?
+
+Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.
```
Comment (kimsauce) [inline docs/alerts/monitors/settings.md:123]:
```suggestion
Tags will appear on all future alerts triggered by the same monitor. Tags will be applied to any past alerts triggered by this monitor.
```

File: docs/alerts/monitors/settings.md (hunk 8 above)
Comment (jinkane) [inline docs/alerts/monitors/settings.md:125]:
```suggestion
To view the tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> You can click on the tab with bell icon to visit the alert list page. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Alert List**. You can also click the **Go To...** menu at the top of the screen and select **Alert List**. Kanso-->
```

File: docs/alerts/monitors/settings.md (hunk 9 above)
Comment (jinkane) [inline docs/alerts/monitors/settings.md:123]:
```suggestion
Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor. If tags on a monitor are edited, that will not be reflected on previous alerts that were triggered by this monitor.
```

File: docs/alerts/monitors/settings.md (hunk 8 above)
Comment (kimsauce) [reply docs/alerts/monitors/settings.md:125]: great catch, @jinkane

File: docs/alerts/monitors/settings.md [hunk 10]
```diff
@@ -84,19 +84,21 @@ Note that the same threshold translating functionality supports to [Creating Mon
...
 ## Tags
 
 Adding a **Tag** to a monitor can help you categorize, search, filter, and correlate them with other data. These tags are key/value pairs that allow you to apply additional metadata to your monitors beyond their name and description. Example tag: `team=alerting`, where key is `team` and value is `alerting`.
 
-:::note Limitations
+### Syntax
+
 - Tag keys cannot start with the prefixes `sumo.` or `_`
-- Tag keys must only contain letters, numbers, and/or the symbols `_`, `.`, `/`, `+`, `-`, `@`
+- Tag keys can only contain letters, numbers, and/or the symbols `_`, `.`, `/`, `+`, `-`, `@`
```
Comment (kimsauce) [inline docs/alerts/monitors/settings.md:97]:
```suggestion
- Tag keys must only contain letters, numbers, and/or the symbols `_`, `.`, `/`, `+`, `-`, `@`
```

**Pull request merged**